book/src/appendices/b_migration_notes/cgmath_to_nalgebra.md

```diff
@@ -0,0 +1,245 @@
+# CG Math to NAlgebra
```

Comment:
The case is weird? Why not `cgmath to nalgebra`?

Reply:
It's in a title position, and at my last workplace we used this convention to distinguish written language from code:

* Use capitalized letters for acronyms when used in a sentence (e.g. XML HTTP Parser)
* Capitalize only the first letter when using an acronym in code (e.g. `XmlHttpParser`)

NAlgebra is an odd case &ndash; there's only 1 letter (here's a fun game, what's N?), and I didn't want it to be lonely

Reply:
I can live with any of:

* "`cgmath` to `nalgebra`"
* "CG Math to NAlgebra"
* "CG Math to Nalgebra"

do people have a preference?

Reply:
This has always seemed like a surprisingly touchy subject when it comes to setting style guides. I've often maintained that the name of any given project should be capitalized in a sentence the way that project does it, like (I think) one would in English for products and companies (e.g. `iTunes`, `eBay`). But I'm not a native speaker.

So all lowercase for me :).

Reply:
I think the lowercase code ones are the best since that's the name they're published under.

Reply:
swapped!